topic-operator/src/main/java/io/strimzi/operator/topic/ReplicasChangeHandler.java

```diff
@@ -4,84 +4,73 @@
+import io.strimzi.operator.topic.cruisecontrol.CruiseControlClient.UserTasksResponse;
+import io.strimzi.operator.topic.cruisecontrol.CruiseControlClientImpl;
 
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.TrustManagerFactory;
-
-import java.io.ByteArrayInputStream;
 import java.io.IOException;
-import java.net.URI;
-import java.net.URLEncoder;
-import java.net.http.HttpClient;
-import java.net.http.HttpRequest;
-import java.net.http.HttpResponse;
-import java.security.KeyStore;
-import java.security.cert.Certificate;
-import java.security.cert.CertificateFactory;
-import java.time.Duration;
-import java.time.temporal.ChronoUnit;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
 import java.util.stream.Collectors;
 
 import static io.strimzi.api.kafka.model.topic.ReplicasChangeState.ONGOING;
 import static io.strimzi.api.kafka.model.topic.ReplicasChangeState.PENDING;
-import static io.strimzi.operator.common.CruiseControlUtil.buildBasicAuthValue;
-import static io.strimzi.operator.topic.TopicOperatorUtil.getFileContent;
 import static io.strimzi.operator.topic.TopicOperatorUtil.hasReplicasChange;
 import static io.strimzi.operator.topic.TopicOperatorUtil.topicNames;
 import static java.lang.String.format;
 import static java.nio.charset.StandardCharsets.UTF_8;
-import static java.util.stream.Collectors.groupingBy;
+import static org.apache.logging.log4j.core.util.Throwables.getRootCause;
 
 /**
  * Replicas change handler that interacts with Cruise Control REST API.
  * <br/><br/>
- * The REST endpoints are {@code topic_configuration} to request replication factor changes and {@code user_tasks} 
- * to check the asynchronous execution result. Cruise Control runs one task execution at a time, additional requests 
- * are queued up to {@code max.active.user.tasks}.
- * <br/><br/>
  * At any given time, a KafkaTopic.spec.replicas change can be in one of the following states:
- * <br/>
  * <ul><li>Pending: Not in Cruise Control's task queue (not yet sent or request error).</li>
  * <li>Ongoing: In Cruise Control's task queue, but execution not started, or not completed.</li>
  * <li>Completed: Cruise Control's task execution completed (target replication factor reconciled).</li></ul>
  */
 public class ReplicasChangeHandler {
     private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(ReplicasChangeHandler.class);
-    private static final String USER_TASK_ID_HEADER = "User-Task-ID";
-    private static final long REQUEST_TIMEOUT_SEC = 60;
     
-    private final TopicOperatorConfig config;
-    private final ExecutorService httpClientExecutor;
-    private final ObjectMapper mapper;
+    private final CruiseControlClient apiClient;
```

Comment:
`ccClient` is better imo, because that tells you what it's a client _of_.

Reply:
I just forgot to rename. Thanks for catching this.